sdk/src/main/scala/coop/rchain/sdk/dag/merging/data/Block.scala

```diff
@@ -0,0 +1,11 @@
+package coop.rchain.sdk.dag.merging.data
+
+// Data required from dag node to be able to merge the dag
+trait Block[B, D] {
+  def id: B
+  def height: Long
+  def deploys: Set[D]
+  def parents: Set[B]
+  def seen: Set[B]
+  def fringe: Set[B]
+}
```

Comment:
This represents just data, there is not need to have a trait for that.

If you are inventing some new ways to write Scala code please provide explanation why is it useful and some reference where similar code exists.